answer/shinooooo/JS/02_add/add.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>add</title>
+</head>
+<body>
+  <script>
+    let i = 1;
+    function add() {
+      document.write(`${i++}<br>`);
```

Comment:
今回は処理が簡易で動いているため問題ないと思いますが。

nits: 
- 数値のカウントアップ。
- 実際にDomに表示。
この二つを同時に行うと。ぱっと見では処理がわかりにくくなってしまう時があるかもしれません。

以下のようにシンプルな物を組み合わせた方がわかりやすいかもしれません
```suggestion
i++
document.write(`${i}<br>`)
```
のように分けた方が良いかなとも感じました。